oppia_android_test.bzl

```diff
@@ -1,41 +1,48 @@
-load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
Linter removes this line of code itself

Lint failure we get without this fix
```
oppia_android_test.bzl:8: load: Loaded symbol "artifact" is unused. Please remove it.
To disable the warning, add '@unused' in a comment.
If you want to re-export a symbol, use the following pattern:

    load(..., _artifact = "artifact", ...)
    artifact = _artifact
 (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load)
```

Reply:
LGTM.